js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx

```diff
@@ -0,0 +1,139 @@
+import { Box, Button, FormLabel } from '@chakra-ui/react';
+import { Global } from '@emotion/react';
+import { Form, Formik } from 'formik';
+import React from 'react';
+import { SwitchControl } from '../../../components/form/SwitchControl';
+import { InfoTooltip } from '../../../components/tooltip/InfoTooltip';
+import { getAdminRouteForCurrentDB } from '../../../utils/arangoClient';
+
+async function rebalanceShards(opts: {
+  maximumNumberOfMoves?: number;
+  leaderChanges?: boolean;
+  moveLeaders?: boolean;
+  moveFollowers?: boolean;
+  excludeSystemCollections?: boolean;
+  piFactor?: number;
+  databasesExcluded?: string[]
+}) {
+  const route = getAdminRouteForCurrentDB();
+  try {
+    const res = await route.put("cluster/rebalance", {
+      version: 1,
+      ...opts
+    });
+    const result = res.body.result;
+    if (result.moves.length === 0) {
+      window.arangoHelper.arangoNotification('No move shard operations were performed.');
+    } else {
+      window.arangoHelper.arangoNotification('Rebalanced shards with ' + result.moves.length + ' move operation(s).');
```

Comment:
maybe we can prefer template strings here?

```suggestion
      window.arangoHelper.arangoNotification(`Rebalanced shards with ${result.moves.length} move operation(s).`);
```